public/js/controllers/game.js

```diff
@@ -162,6 +158,24 @@ angular.module('mean.system')
       }
     });
 
+    $scope.$watch('game.userExist', function () {
+      if (game.userExist) {
+        $scope.userExist();
+      }
+    });
+
+    $scope.userExist = function () {
```

Comment:
No need to add this to the scope, the function can stand on its own without been attached to the scope. Better still are you using it outside the controller or in the html template file?